### ISSUE TITLE
Video freezes when you moving the position slider

### ISSUE THREAD

**Issue by thereticle** [otsaloma/gaupol#196]
When you move the cursor to find a certain forward position, the video freezes.
Sometimes the situation is solved by reloading the video, but in general you need to close the application and start again.
The problem occurs with both MPlayer and VLC.
O.S. Ubuntu 22.04
Gaupol v. 1.10.1

**Comment by otsaloma** [comment]
Thanks for reporting the issue.

To clarify, by "cursor" do you mean the position slider in the integrated video player (picture below)?

![Screenshot from 2022-06-26 22-38-17](https://user-images.githubusercontent.com/237265/175831033-ff93b2b2-6da7-420b-a883-97a5106c9206.png)

MPlayer and VLC are only for the "preview" functionality ("Preview" from the "Tools" menu). Those just launch the video player and the players don't interact with Gaupol in any way. The integrated video player uses GStreamer.

**Comment by thereticle** [comment]
Yes, I mean the slider. 
When I move the slider , the app freeze and no further interaction is possibile until I restart it.
Maybe the issue is related to GStreamer.

**Comment by otsaloma** [comment]
Thanks. I've never seen this kind of a problem before and I couldn't reproduce it as I tried. There two likely possibilities

1. It's related to the particular file or file format. If only a single file causes problems, it could be in some way broken and thus seeking not working properly. Or if it's a single file format (or video encoder) causing problems, it could be related to how GStreamer handles that. The only thing that comes to mind is that you could check that you have the following packages all installed. If you installed Gaupol via the package manager, you should have them though.

```
gstreamer1.0-libav
gstreamer1.0-plugins-bad
gstreamer1.0-plugins-good
gstreamer1.0-plugins-ugly
```

2. There could be some kind of a problem in how Gaupol handles the playback state, maybe something like one action (looping playback, seeking, etc.) started before the previous has finished. If there's some particular sequence of actions by which you can reliably reproduce the problem, then let me know.

**Comment by thereticle** [comment]
Oh Oh...
The problem has come back and it seems to happen when I have a browser open.
I normally use Firefox, but I've also tried with Chrome. With both of them, after a while they are open, Gaupol freezes.
Closing Gaupol and the browsers and reopening only Gaupol, everything works.
So there must be some interaction between these software.
I really don't know what kind of interaction...
I reported it just for your info.
Thanks again
